spec/requests/user_pages_spec.rb

```diff
@@ -1,416 +1,416 @@
+        fill_in "Notes", with: "Here are some notes"
+        select reviewing_group.name, from: "Reviewing group"
+        find("#token-input-user_associate_consultant_attributes_coach_id").set(coach.id)
+
+        click_button 'Create'
+
+        current_path.should eql users_path
+        page.should have_selector('.flash-success', text: 'User "Roberto Glob" was successfully created with 6-Month, 12-Month, 18-Month, and 24-Month reviews.')
+
+        new_user = User.last
+        new_user.associate_consultant.reviews.size.should == 4
+        id_selector = new_user.email.slice(0..(new_user.email.index('@'))-1)
+        page.should have_selector("tr##{id_selector} .fa-book")
+
+        visit user_path(new_user)
+
+        new_ac = new_user.associate_consultant
+        page.should have_selector("#isAC", text: 'Yes')
+        page.should have_selector("#hasGraduated", text: 'No')
+        page.should have_selector("#notes", text: new_ac.notes)
+        page.should have_selector("#reviewing-group", text: new_ac.reviewing_group)
+      end
+    end
+
+    describe "as a non-admin user" do
+
+      it "cannot create new users" do
+        sign_in create(:user)
+        visit new_user_path
+        current_path.should == root_path
+        page.should have_selector('.flash-alert', text: "You are not authorized to access this page.")
+      end
+    end
+  end
+
+  describe "SHOW page" do
+    let (:admin) { create(:admin_user) }
+    let!(:user) { create(:user) }
+
+    describe "as an admin" do
+      before do
+        sign_in admin
+        visit user_path(user)
+      end
+
+      it { should have_content(user.name) }
+      it { should have_content(user.email) }
+      it { should have_content("Admin No") }
+
+      it "links to edit page" do
+        click_link "Edit"
+        current_path.should == edit_user_path(user)
+      end
+
+      it "links to users index page" do
+        page.find(".diet.button", text: "Manage Users").click
+        current_path.should == users_path
+      end
+    end
+
+    describe "as a non-admin" do
+      before do
+        sign_in user
+        visit user_path(user)
+      end
+
+      it "is accessible if visiting own page" do
+        current_path.should == user_path(user)
+      end
+    end
+  end
+
+  describe "INDEX page" do
+    let!(:user) { create(:user, name: 'Andy', email: 'andy@example.com') }
+    let!(:admin) { create(:admin_user) }
+
+    describe "as an admin" do
+      before do
+        sign_in admin
+        visit users_path
+      end
+
+      it { should have_selector('h1', text: 'Manage Users') }
+      it { should have_selector('th', text: 'Name') }
+      it { should have_selector('th', text: 'Email') }
+
+      it "has user fields displayed in table", js:true do
+        should have_selector('td', text: 'Andy')
+        should have_selector('td', text: 'andy@example.com')
+        should have_selector('td', text: admin.name)
+        should have_selector('td', text: admin.email)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.